stdlib/public/runtime/Private.h

```diff
@@ -183,6 +183,8 @@ namespace swift {
                              void *data);
   uint8_t *_swift_getSectionDataPE(void *handle, const char *sectionName,
                                    unsigned long *sectionSize);
+#endif
+#if defined(__CYGWIN__)
   void _swift_once_f(uintptr_t *predicate, void *context,
```

Comment:
It is the code after `if (Builtin.ID == BuiltinValueKind::Once)` in `lib/IRGen/GenBuiltin.cpp`.